File: Submissions/#0386_div2/B/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int n;
  cin >> n;

  string s;
  cin >> s;

  string answer = s.substr(n - 1, 1);

  for (int i = 2; i <= n; i++) {
    string s_ = s.substr(s.size() - i, 1);
    answer.insert(answer.size() / 2, s_);
  }

  cout << answer << endl;

}

File: Submissions/#0386_div2/B/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    String s = sc.next();
    sc.close();

    StringBuilder answer = new StringBuilder(s.substring(n - 1));

    for (int i = 2; i <= n; i++) {
      String s_ = s.substring(n - i, n - i + 1);
      answer.insert(answer.length() / 2, s_);
    }

    System.out.println(answer.toString());

  }

}